woocommerce/woocommerce
# [Experimental] Product Filter - Reintroduce Wrapper Block

### Changes proposed in this Pull Request:

It was determined after some product review feedback that it makes more sense from blocks paradigm perspective to reintroduce a wrapper block for the new interactivity filter blocks.

To do that I've migrated the product filters block to become the wrapper block.

Closes https://github.com/woocommerce/woocommerce/issues/43544 and https://github.com/woocommerce/woocommerce/issues/43545

Also we decided to rename the blocks slightly to "Product Filter" from "Product Filters". 



### How to test the changes in this Pull Request:


1. Create page or post with product collection (beta) and add some or all of the filter blocks (e.g. `Product Filters: Price`)
2. Smoke test the filters
3. Test that when filters are not relevant they are hidden (including their respective headings). e.g. if you filtered down to products with no reviews then the whole review filter and heading should be hidden.
4. Add one of the old product filters e.g. `Filter by Rating`. In the inspector, attempt to use the "upgrade" button to upgrade it. Also attempt to use "downgrade" from the newly upgraded block.
5. in CI for this PR ensure that the e2e tests for this feature pass



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

[Experimental] Reintroduce a wrapper block for the interactivity powered filter blocks.

#### Comment 

</details>
